qiita_pet/uimodules/search_sample_list.py

```diff
@@ -0,0 +1,12 @@
+from .base_uimodule import BaseUIModule
+
+
+class SearchSampleList(BaseUIModule):
+    def render(self, study_id, meta_headers,
+               proc_data_dict, proc_data_samples):
+        selected = not meta_headers
+
+        return self.render_string(
+            "search_sample_list.html", meta_headers=meta_headers, sid=study_id,
+            proc_data_dict=proc_data_dict, proc_data_samples=proc_data_samples,
+            selected=selected)
```

Comment:
is this also done in the template? It doesn't seem necessary to keep track of it separately in the handler since it is always just the boolean NOT of meta_headers. If this *is* necessary, please add a comment, since I was confused by it and others might also be.